doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -26,6 +26,8 @@ Traceability to Reference Model.
 <a name="2.3"></a>
 ## 2.3 Architecture and OpenStack Requirements.
 
+"Architecture" in this chapter refers to NFVI + VIM (as specified in Reference Model Chapter 3) and consists of the Physical resources, virtual and container resources, and Software Management Layer (Hypervisor, Container runtime, virtual or container Orcehstrator(s), Operating System).
```

Comment:
I think this list of what NFVI and VIM consist of is not needed here and also incomplete (says nothing about vSwitch or vRouter for example).

Reply:
@CsatariGergely I think this is the language that we had discussed on one of the calls. Please suggest alternate language. Thanks

Reply:
I would use only this '"Architecture" in this chapter refers to NFVI + VIM (as specified in Reference Model Chapter 3)' and delete the rest.